docs/user_guide/plotting.md

```diff
@@ -1,11 +1,341 @@
+requested processes will be considered. When only datasets are given, all processes in the config
+will be considered.
+The ```processes``` parameter can be used to change the order of processes in the stack and the legend
+(try for example ```--processes st,tt``` instead) and to further distinguish between sub-processes
+(e.g. via ```--processses, tt_sl,tt_dl,tt_fh```).
+
+:::{dropdown} Note! Do not add the same dataset via multiple processes!
+At the time of writing this documentation, there is still an issue present that histograms corresponding
+to a dataset can accidentally be used multiple times. For example, when adding ```--processes tt,tt_sl```,
+the events corresponding to the dataset ```tt_sl_powheg``` will be displayed twice in the resulting
+plot.
+:::
+
+
+## Customization of plots
+
+There are many different parameters implemented that allow customizing the style of a plot. A short
+description to all plotting parameters is given in the [Plotting tasks](../task_overview/plotting.md).
+In the following, a few exemplary task calls are given to present the usage of our plotting parameters.
+
+Per default, the PlotVariables1D task creates one plot per variable with all Monte Carlo backgrounds being included
+in a stack and data being shown as separate points. The bottom subplot shows the ratio between signal
+and all processes included in the stack and can be disabled via the ```--skip_ratio``` parameter.
+To change the text next to the label, you can add the ```--cms-label``` parameter.
+To compare shapes of multiple processes, you might want to plot each process separately as one line.
+To achieve this, you can use the ```unstack``` option of the ```--process-settings``` parameter. This
+parameter can also be used to change other attributes of your process instances, such as color, label,
+and the scale. To better compare shapes of processes, we can normalize each line with the
+```--shape-norm``` parameter. Combining all the previously discussed parameters might lead to a task
+call such as
+
+```shell
+law run cf.PlotVariables1D --version v1 --processes tt,st --variables n_jet,jet1_pt \
+    --skip-ratio --shape-norm --cms-label simpw \
+    --process-settings "tt,unstack,color=#e41a1c:st,unstack,label=Single Top"
+```
+
+to produce the following plot:
+
+::::{grid} 1 1 2 2
+:::{figure} ../plots/cf.PlotVariables1D_tpl_config_analy__1__0191de868f__plot__proc_2_a2211e799f__cat_incl__var_jet1_pt__c1.pdf
+:width: 100%
+:::
+
+:::{figure} ../plots/cf.PlotVariables1D_tpl_config_analy__1__0191de868f__plot__proc_2_a2211e799f__cat_incl__var_n_jet__c1.pdf
+:width: 100%
+:::
+::::
+
+
+Parameters that only contain a single value can also be passed via the ```--general-settings```.
+We can also change the y-scale of the plot to a log scale by adding ```--yscale log``` and change some
+properties of specific variables via the ```variable-settings``` parameter. An exemplary task call
```

Comment:
I think explaining a little more what we want to do in this example might be instructive. Maybe sth like

```suggestion
properties of specific variables via the ```variable-settings``` parameter. For example, we might want to create the plots of our two obserables in one call, but would like to try out a rebinned version of `jet1_pt` that only consideres every tenth bin. A corresponding task call
```